repository/mock_repo.go

```diff
@@ -82,6 +82,17 @@ func (r *mockRepoForTest) RmConfigs(keyPrefix string) error {
 	return nil
 }
 
+func (r *mockRepoForTest) ListRemotes() (map[string]string, error) {
+	remotes := make(map[string]string)
+	for key, val := range r.config {
+		if _, ok := remotes[key]; ok {
```

Comment:
this would match much more than just the remotes. Have a look at the output of `git config --list`.